### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by maaloum**
In this project I : 

- Add a .gitignore file.
- Set up a linter for HTML and CSS
- Add a meaningful README file 
- Add 3 files :
    - A .html file that includes a header with the text "Hello Microverse!". Using h1 tag.
    - A .css file that includes one class with styles for the header to modify color.
    - A test.md file and ignored by git".

### CODE REVIEW

File: html-css/.github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
```
Comment (topeogunleye) [inline html-css/.github/workflows/linters.yml]: - [ ] Your linters should be set up in your root folder for your linter results to show on Github. KIndly move your linters to your root folder.
- [ ] Kindly also ensure that all your linter results are green i.e Linter should report NO errors.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (topeogunleye) [inline .gitignore]: - [ ] Kindly add a '/' in front on `node_modules` to specify that it's a directory, and also add `test.md` to your `.gitignore` file

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @maaloum,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Followed Github flow :heavy_check_mark:

> ## (Changes Required) Aspects to improve: 
> *  ### Please do the following:
>- [x] Please kindly add a more descriptive PR title and description.
> See a demo below:

https://user-images.githubusercontent.com/52934128/176308229-e31b2c8d-110b-4823-a2a8-31c1b7d7c73d.mp4


>- [ ] Please refer to this article: please read it and apply it in your work from now on: [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md);

>- [x] Kindly adapt your README template for your project by replace the template info with your `Hello-Microverse project` info.

>- [x] From now on, I kindly suggest that you please make use of imperative mood in the subject line for your commit messages.
:x: Wrong
image)
> ### Good Commit Messages:
>- [x]  Add fix for dark mode toggle state
>- [x] Update README file.
>- [x] Add styles to h1.
>


> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thank you Microverse
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (Nemwel-Boniface) [inline README.md:45]: [OPTIONAL] - Kindly make sure to include an MIT.md in your project. If you may be in doubt, have a look at [this](https://github.com/microverseinc/readme-template/blob/master/MIT.md) example MIT.md this way your documentation will be 100% complete :sunglasses: .

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @maaloum  :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making the changes that were requested by the previous reviewer.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 
## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**